front/pages/w/[wId]/ds/[name]/settings.tsx

```diff
@@ -553,6 +554,7 @@ function ManagedDataSourceSettings({
 
   const { total } = useDocuments(owner, dataSource, 0, 0);
 
+  // @todo: When updating a permission we lost a potential suffix on the connectionId name. Fix it?
```

Comment:
I think this one is quite important as it would break for workspace that have more than a single WS for a given connector type

Reply:
Actually I think we can just kill suffixes??